packages/go/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@vercel/go",
-  "version": "2.0.15",
+  "version": "2.0.16",
```

We publish as a separate step. I'll follow up with more details in the slack thread.

```suggestion
  "version": "2.0.15",
```